### PR TITLE
Updates for Ubuntu 20.04

### PR DESCRIPTION
**Pull request by enoch85**
I will add more if I find any. Following the guide now

---
This change is [Reviewable](https://reviewable.io/reviews/flurdy/flurdy.com-docs/15)

### CODE REVIEW

File: postfix/edition15.html [hunk 1]
```diff
@@ -1401,20 +1406,25 @@ <h5>Configuration</h5>
...
-		But we need to configure it, so:
-		<code>sudo vi /etc/mysql/my.cnf</code>
-		In previous versions you needed to comment out this line
-		<code class="document"><span class="comment">#skip-networking</span></code>
-		However in today's file the default is to bind the
-		address to localhost, which is fine.
+		Next is to create a specific config MySQL config file.
+		Before we start adding configuring options, please double check that
+		the bind-adress value is <b>not</b> hashed out:
+		<code>
```
Comment (enoch85) [inline postfix/edition15.html:1412]: @flurdy  Please check the formatting here, a little bit uncertain.

**Comment by enoch85** [comment]
cc @flurdy This is ready for review!

**Comment by flurdy** [comment]
This looks good. Will double check :)

File: postfix/edition15.html [hunk 2]
```diff
@@ -1184,7 +1188,7 @@ <h5>SSH only</h5>
 sudo vi /etc/shorewall/zones</code>
 		Add the firewall if not there and the internet as a zone, and the optional VPC if you need one.
 		<code class="document">fw	firewall
-net     ipv4d
+net     ipv4
```
Comment (flurdy) [inline postfix/edition15.html:1191]: Another typo? Thanks :flushed:

**Comment by flurdy** [comment]
So basically update distro to focal. Update Shorewall to new config examples. Update MySQL grants and separate mailconfmysql.cnd file. All seems sensible.

**Comment by enoch85** [comment]
> Update Shorewall to new config examples

Yeah, it could be improved even more, but I didn't have time to look into the proper way to add interfaces in `/etc/shorewall/interfaces`. Newer shorewall use `FORMAT?2` but `FORMAT?1` is still allowed, (though deprecated) so I went with that.

Other than that, yes you are correct. This should be good to go!

**Comment by enoch85** [comment]
@flurdy Can you please merge this? I need to redo everything again and this would help me if merged.

Thanks!

**Comment by flurdy** [comment]
Yup will do

File: postfix/edition15.html (hunk 1 above)
Comment (enoch85) [reply postfix/edition15.html:1412]: @flurdy  As I suspected, the formatting was wrong.

image

Maybe you can fix it in another PR?

Thanks!

**Pull request merged**